qa/tasks/cephadm.py

```diff
@@ -566,22 +592,9 @@ def ceph_mgrs(ctx, config):
     fsid = ctx.ceph[cluster_name].fsid
 
     try:
-        nodes = []
-        daemons = {}
-        for remote, roles in ctx.cluster.remotes.items():
-            for mgr in [r for r in roles
-                        if teuthology.is_type('mgr', cluster_name)(r)]:
-                c_, _, id_ = teuthology.split_role(mgr)
-                if c_ == cluster_name and id_ == ctx.ceph[cluster_name].first_mgr:
-                    continue
-                log.info('Adding %s on %s' % (mgr, remote.shortname))
-                nodes.append(remote.shortname + '=' + id_)
-                daemons[mgr] = (remote, id_)
-        if nodes:
-            _shell(ctx, cluster_name, remote, [
-                'ceph', 'orch', 'apply', 'mgr',
-                str(len(nodes) + 1) + ';' + ';'.join(nodes)]
```

Comment:
any idea why mgr was deployed on +1 whereas the other services are not?

Reply:
remember the old scheduler algorithm?